scapy/utils.py

```diff
@@ -1437,6 +1438,9 @@ def tcpdump(pktlist, dump=False, getfd=False, args=None,
         prog = [conf.prog.tcpdump]
     elif isinstance(prog, six.string_types):
         prog = [prog]
+    if not WINDOWS and not _check_tcpdump():
```

Comment:
Also also, it's overkill to call `_check_tcpdump` (
```python
with open(os.devnull, 'wb') as devnull:
    try:
        proc = subprocess.Popen([conf.prog.tcpdump, "--version"],
                                stdout=devnull, stderr=subprocess.STDOUT)
     except OSError:
        return False
```
) on each opened socket. There should be a cache or something